lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -0,0 +1,154 @@
+                                    valueAttribute: "level_of_education",
+                                    options: fields_data['level_of_education']['options'],
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.DropdownFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Gender"),
+                                    valueAttribute: "gender",
+                                    options: fields_data['gender']['options'],
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.DropdownFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Year of Birth"),
+                                    valueAttribute: "year_of_birth",
+                                    options: fields_data['year_of_birth']['options'],
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.DropdownFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Country"),
+                                    valueAttribute: "country",
+                                    options: fields_data['country']['options'],
+                                })
+                            },
+                            {
+                                view: new AccountSettingsFieldViews.DropdownFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Preferred Language"),
+                                    valueAttribute: "language",
+                                    helpMessage: gettext("If your preferred language isn't available as an edX language, you can indicate your preferance here."),
+                                    options: fields_data['preferred_language']['options'],
+                                })
+                            },
+                        ]
+                    },
+                    {
+                        title: gettext("Connected Accounts"),
+                        fields: [
+                            {
+                                view: new AccountSettingsFieldViews.LinkFieldView({
+                                    model: accountSettingsModel,
+                                    title: gettext("Facebook"),
+                                    valueAttribute: 'auth-facebook',
+                                    linkTitle: gettext("Link"),
+                                    helpMessage: gettext("Under construction."),
```

Comment:
Is this placeholder text - in other words, will this section of the page be working for MVP? 

Assuming that this section is visible but not completely functional at MVP, I'd recommend changing the "Under construction" text to "Coming soon" (with no period).